src/utils/loaders/getCityFeed.test.js

```diff
@@ -1,4 +1,27 @@
-const problems = require('./getCityFeed')
-test('first test to Add function', () => {
-    expect(problems.add(1,2)).toEqual(3)
-})
\ No newline at end of file
+import { getCityFeed } from './getCityFeed'
+
+describe('getCityFeed function', () => {
+  // Mocking fetch
+  global.fetch = jest.fn(() =>
+    Promise.resolve({
+      json: () => Promise.resolve({ mockData: 'mockValue' }),
+    })
+  )
+
+  beforeEach(() => {
+    fetch.mockClear()
+  })
+
+  it('should call fetch with the correct URL', async () => {
+    const token = 'mockToken'
+    const expectedURL = `https://api.waqi.info/feed/here/?token=${token}`
+    await getCityFeed(token)
+    expect(fetch).toHaveBeenCalledWith(expectedURL)
+  })
+
+  it('should return JSON data from the response', async () => {
+    const token = 'mockToken'
+    const result = await getCityFeed(token)
+    expect(result).toEqual({ mockData: 'mockValue' })
```

Comment:
can you test with some data that close to real data, at least structure wise

Reply:
test suite updated